docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-costs.md

```diff
@@ -1,17 +1,29 @@
 ---
 title: Cluster costs
 linkTitle: Cluster costs
-description: YugabyteDB Managed cluster configuration costs.
-headcontent: YugabyteDB Managed cluster configuration costs
+description: YugabyteDB Aeon cluster configuration costs.
+headcontent: Plan-based cluster pricing
```

Comment:
@ddhodge pricing is for the account, not per cluster or cluster based. Can we remove this sub-heading?